src/main/webapp/i18n/de/error.json

```diff
@@ -47,6 +47,40 @@
         "submissionPolicyToggleFailedPolicyNotExist": "Für diese Programmieraufgabe konnte die Abgaberichtlinie nicht (de)aktiviert werden, da die Programmieraufgabe keine Abgaberichtlinie hat. ",
         "submissionPolicyActiveNull": "Abgaberichtlinien müssen aktiviert oder deaktiviert sein. Dieser Status darf nicht null sein.",
         "submissionPolicyUpdateFailedPolicyNotExist": "Die Abgaberichtlinie konnte nicht gespeichert werden, da die Programmieraufgabe keine Abgaberichtlinie hat!",
-        "oneActivePlagiarismCheck": "Bitte warte bis die aktuelle Plagiatskontrolle in diesem Kurs abgeschlossen ist, bevor du eine weitere startest."
+        "oneActivePlagiarismCheck": "Bitte warte bis die aktuelle Plagiatskontrolle in diesem Kurs abgeschlossen ist, bevor du eine weitere startest.",
+        "internalServerError": "Interner Serverfehler",
+        "server.not.reachable": "Server ist nicht erreichbar",
+        "url.not.found": "Nicht gefunden",
+        "NotNull": "Feld {{fieldName}} darf nicht leer sein!",
+        "size": "Feld {{fieldName}} erfüllt nicht die minimalen/maximalen Längenanforderungen!",
+        "userexists": "Login bereits vergeben!",
+        "emailexists": "E-Mail wird bereits verwendet!",
+        "idexists": "Ein neuer {{entityName}} kann noch keine ID haben",
+        "ambiguousContext": "Ein neuer {{entityName}} muss einen eindeutigen Kontext haben",
+        "missingname": "Ein neues {{entityName}} braucht einen Namen",
```

Comment:
I did not find any use of that key inside of Artemis. Is this also uses by jhipster or something similar?

Reply:
Some error messages are implicit and handled automatically - there is no explicit call on the client needed (e.g. see delete user)

Reply:
![image](https://user-images.githubusercontent.com/60005702/175989472-8c7338d8-6a9d-4d82-bb0b-93e5fc3a5f8e.png)

:thinking: